so/challenges/default.json

```
@@ -50,4 +50,111 @@
+      "default": false,
+      "schedule": [
+        {
+          "operation": "delete-index"
+        },
+        {
+          "operation": {
+            "operation-type": "create-index",
+            "settings": {{index_settings | default({}) | tojson}}
+          }
+        },
+        {
+          "name": "check-cluster-health",
+          "operation": {
+            "operation-type": "cluster-health",
+            "index": "so",
+            "request-params": {
+              "wait_for_status": "{{cluster_health | default('green')}}",
+              "wait_for_no_relocating_shards": "true"
+            },
+            "retry-until-success": true
+          }
+        },
+        {
+          "operation": "index-append",
+          "warmup-time-period": 120,
+          "clients": {{bulk_indexing_clients | default(8)}},
+          "ignore-response-error-level": "{{error_level | default('non-fatal')}}"
+        },
+        {
+          "operation": {
+            "operation-type": "force-merge",
+            "request-timeout": 7200
+          }
+        },
+        {
+          "name": "wait-until-merges-finish",
+          "operation": {
+            "operation-type": "index-stats",
+            "index": "_all",
+            "condition": {
+              "path": "_all.total.merges.current",
+              "expected-value": 0
+            },
+            "retry-until-success": true,
+            "include-in-reporting": false
+          }
+        },
+        {
+          "name": "delete-transform-group-by-user",
+          "operation": {
+            "operation-type": "delete-transform",
+            "transform-id": "transform-group-by-user",
+            "force": true
+          }
+        },
+        {
+          "operation": "create-transform-group-by-user"
+        },
+        {
+          "name": "start-transform-group-by-user",
+          "operation": {
+            "operation-type": "start-transform",
+            "transform-id": "transform-group-by-user"
+          }
+        },
+        {
+          "name": "wait-for-transform-group-by-user",
+          "operation": {
+            "operation-type": "wait-for-transform",
+            "transform-id": "transform-group-by-user",
+            "transform-timeout": 3600,
+            "timeout": "60m"
+          }
+        },
+        {
+          "name": "delete-transform-group-by-hour",
+          "operation": {
+            "operation-type": "delete-transform",
+            "transform-id": "transform-group-by-hour",
+            "force": true
+          }
+        },
+        {
+          "operation": "create-transform-group-by-hour"
+        },
+        {
+          "name": "start-transform-group-by-hour",
+          "operation": {
+            "operation-type": "start-transform",
+            "transform-id": "transform-group-by-hour"
+          }
+        },
+        {
+          "name": "wait-for-transform-group-by-hour",
+          "operation": {
```

If the point is to measure duration here, you will want 
```suggestion
          "include-in-reporting": true,
          "operation": {
```